packages/browser/src/loader.js

```diff
@@ -58,6 +58,7 @@
     // come out in the wrong order. Because of that we don't need async=1 as GA does.
     // it was probably(?) a legacy behavior that they left to not modify few years old snippet
     // https://www.html5rocks.com/en/tutorials/speed/script-loading/
+    var _currentScriptTag = _document.getElementsByTagName(_script)[0];
```

Comment:
what if it's `null`?

Reply:
It cannot be null, as it's our own loader's script tag.

Reply:
Unless it's some odd edge-case scenario in non-browser environment, but you shouldn't use a loader then anyway.